astropy/io/ascii/tests/test_cds.py

```diff
@@ -547,7 +547,7 @@ def test_write_skycoord_with_format():
     ):
         t.write(out, format='ascii.mrt', formats={'RAs': '05.2f', 'DEs': '04.1f'})
 
-    lines = out.getvalue().splitlines()
+    lines = [line.rstrip() for line in out.getvalue().splitlines()]
```

Comment:
@taldcroft I generally agree with your emailed suggestion to accept trivial code changes in these PRs, especially where already committed. This also seemed an inconspicuous change merely getting the test to work again.
But looking at this again today I am having second thoughts – if I recall correctly, the current behaviour of the MRT writer is to pad all header lines to the width of the longest content (non-blank) string. This does not actually seem to be mandated by the MRT standard, so in principle the writer would be free to just put out the stripped lines.
But the current `astropy.io.ascii` functionality is to create a header padded to constant line widths (I think implemented to make parsing more convenient), and by now people may have come to rely on that property. The previous test was implicitly also testing this, and I'd feel better if we kept it that way and protect the blank space in `exp_output` from black instead, as @WilliamJamieson initially proposed.

Reply:
Ah, I thought that last point [had been discussed](https://github.com/astropy/astropy/pull/12302#issuecomment-955086901) before!

Reply:
Fixed